src/navigation/underline-nav.scss

```diff
@@ -31,7 +31,8 @@
     }
   }
 
-  &.selected {
+  &[role=tab][aria-selected=true],
+  &.selected { // TODO@14.0.0: remove &.selected
```

Comment:
Removing `.selected` in `14.0.0` might be a bit too early because there are probably a few use-cases that use the `UnderlineNav` without [`tab-container`](https://github.com/github/tab-container-element), like [this one](https://github.com/github/github/blob/f4cb263b82334fa4a7a42eafac2c080ad5f029e0/app/views/orgs/team_discussions/index.html.erb#L233).

Reply:
> should be pretty straightforward.

😬 ok, let's try... in worst case, there is always the `deprecated` folder that could be used in emergencies.

Also, 👍 for adding `aria-current`. How I understood this [article](https://www.stefanjudis.com/blog/aria-selected-and-when-to-use-it/).

- Use `aria-selected=true` for navs that toggle some content.
- Use `aria-current=page` for navs that link to another page (change the URL).

```scss
&.selected,  // TODO@14.0.0: remove &.selected
&[aria-selected=true],
&[aria-current]:not([aria-current=false]) {
```